ACMILabs/interactive-label
# #74 max 15 images

Resolves #74

This achieves a max of 15 images in the gallery by only caching the first 15 images from the serialized label's image list.

### Acceptance Criteria
Replace acceptance criteria with the acceptance criteria in the ticket or check the box if none.
- [x] Only displays 15 images in the gallery
- [x] Doesn't waste time downloading images we're not displaying :)

### Relevant design files
Add a links to the relevant design files or leave as none.
* None

### Testing instructions
Add a list of steps required to test the feature.
1. Switch to MI-02-LR03 by settings XOS_PLAYLIST_ID=43 in config.env
2. Click the Jane Conquest Narrative slides (pastel colours, upper left)
3. See that only 15 images are displayed.

### DoD
For requester to complete:
